rpcs3/Gui/GSFrame.h

```diff
@@ -39,4 +39,4 @@ class GSFrame : public wxFrame, public GSFrameBase
 
 private:
 	DECLARE_EVENT_TABLE();
-};
\ No newline at end of file
+};
```

Comment:
Because someone more experienced than you said so.
For more information please refer to: https://en.wikipedia.org/wiki/Argument_from_authority

PS: Otherwise, read this: http://stackoverflow.com/a/729795